packages/core/components/DraggableComponent/index.tsx

```diff
@@ -18,6 +18,7 @@ import { Loader } from "../Loader";
 import { ActionBar } from "../ActionBar";
 import { DefaultOverride } from "../DefaultOverride";
 import { useLoadedOverrides } from "../../lib/use-loaded-overrides";
+import { getPermissions } from "../../lib/get-permissions";
```

Comment:
@chrisvxd, is this the right approach here?  Using the lib not the usePuck hook?